pkg/xray/provider.go

```diff
@@ -133,3 +145,24 @@ func sendUsageRepo(restyBase *resty.Client, terraformVersion string) (interface{
 	}
 	return nil, nil
 }
+
+func checkArtifactoryLicense(client *resty.Client) error {
+
+	type License struct {
+		Type         string `json:"type"`
+		ValidThrough string `json:"validThrough"`
+		LicensedTo   string `json:"licensedTo"`
+	}
+
+	license := License{}
+	_, err := client.R().SetResult(&license).Get("/artifactory/api/system/licenses/")
+	if err != nil {
+		return err
+	}
+
+	if !strings.Contains(license.Type, "Enterprise") {
```

Comment:
There are actually a few types that might work. Currently there is an outstanding ticket in RT about this

Reply:
Will implement on the next PR